$NetBSD: patch-Makefile,v 1.1 2023/07/12 20:46:24 nia Exp $

Append, don't set LDFLAGS, to ensure the environment is honoured.

--- Makefile.orig	2023-07-12 20:41:37.017940349 +0000
+++ Makefile
@@ -50,11 +50,9 @@ PREFIX=/usr/pkg
 INSTALLDIR=	install -d -m 755
 INSTALLBIN=	install -m 555
 INSTALLMAN=	install -m 444
-CC=		gcc
-CFLAGS=		-Wall -W -I/usr/X11R6/include -O
 # The following line is what I use during development
 #CFLAGS:=	$(CFLAGS) -Werror -g 
-LDFLAGS=	-L/usr/X11R6/lib -lXpm -lXext -lX11 -lm -lcurl
+LDFLAGS+=	-lXpm -lXext -lX11 -lm -lcurl
 DOCS=		wmget.1
 
 VERSION:=	$(shell grep '\#define WMGET_VERSION ' wmget.h \
